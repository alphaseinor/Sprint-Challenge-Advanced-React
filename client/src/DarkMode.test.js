import React from 'react';
import { render, cleanup } from '@testing-library/react';
import NavBar from './Components/NavBar';

afterEach(cleanup);

it('renders', () => {
  const {asFragment} = render(<NavBar dark=".dark-mode" />)
  expect(asFragment()).toMatchSnapshot();
});