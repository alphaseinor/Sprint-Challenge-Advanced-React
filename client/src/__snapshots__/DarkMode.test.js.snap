// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <h1>
      Womens World Cup
    </h1>
    <div
      class="dark-mode__toggle"
    >
      <div
        class="toggle"
      />
    </div>
  </nav>
</DocumentFragment>
`;
